temporal-remote-data-encoder/README.md

```diff
@@ -0,0 +1,31 @@
+# Temporal Remote Data Encoder module
+
+## What problems does Remote Data Encoder (RDE) solves?
+
+- Allows to reuse complicated encryption logic written once between different languages
+- Allows tctl to encode payloads for tctl workflow start and Temporal WebUI to decode encrypted payloads
+- Allows creation of a service that has an access to encryption keys and performing the encryption/decryption instead of allowing developer workstations to access the keys directly.
```

Comment:
```suggestion
- Allows reuse of complicated encryption logic written once between different languages
- Allows `tctl` to encode payloads for `tctl workflow start` and Temporal WebUI to decode encrypted payloads
- Allows creation of a service that has an access to encryption keys for performing the encryption/decryption instead of a developer workstation or service accessing the keys directly.
```